src/auth0/handlers/pages.js

```diff
@@ -19,7 +19,7 @@ export const schema = {
     properties: {
       name: { type: 'string', enum: supportedPages },
       html: { type: 'string', default: '' },
-      url: { type: 'string', default: '' },
+      url: { type: 'string' },
```

Comment:
This was changed because the default was adding a `url` property in dump that was not expected in `deploy-cli`. Not related to the PR changes